Coordinator/ArchitectureCoordinator.swift

```diff
@@ -0,0 +1,68 @@
+//
+//  ArchitectureCoordinator.swift
+//  iOSPracticals
+//
+//  Created by Yagnik Bavishi on 13/04/22.
+//
+
+import Foundation
+import UIKit
+
+class ArchitectureCoordinator: Coordinator {
+    
+    var navController: UINavigationController?
+    let userDefaults = UserManager()
+    
+    init(_ navigationController: UINavigationController) {
+        navController = navigationController
+    }
+    
+    func start() {
+        userDefaults.isUserLogin ? startUserListVC() : startMVVMArchitecture()
+    }
+    
+    func startArchitectureStoryBoard() {
```

Comment:
instead of start you can use goToArchitectureStoryBoard or MoveToArchitectureStoryboards
or you can just simply write like for example goToDashboard or moveToDashboard